### ISSUE TITLE
[SkipToContent]: does not meet WCAG AA foreground/background contrast minimum

### ISSUE THREAD

**Issue by jendowns** [carbon-design-system/gatsby-theme-carbon#570]
The contrast between the `SkipToContent` component's link text and the background color does not meet WCAG AA foreground/background contrast minimum.

On every page of the Demo site, for example, you can see the following DAP violation associated with the `SkipToContent` component:

![Screen Shot 2019-11-26 at 9 35 41 AM](https://user-images.githubusercontent.com/9057921/69648029-2563e600-1030-11ea-9b2a-5c9ce0807c11.png)

## What WCAG 2.1 checkpoint does the issue violate?
[Checkpoint 1.4.3 - Contrast minimum](https://www.ibm.com/able/guidelines/ci162/contrast.html)

**Comment by jendowns** [comment]
I can propose a fix for this shortly 👍
